bin/run_integration

```diff
@@ -49,7 +49,7 @@ export COMPOSE_PROJECT_NAME
 # Validate the required scripts exists in the TEST_SUBDIR
 
 project_dir=$PWD
-test_dir="${project_dir}/test/${non_flag_args[0]}"
+test_dir="$(find "${project_dir}"/test -type d -name "${non_flag_args[0]}")"
```

Comment:
Small thing and I know this is existing code, but as a general rule when editing a file for a new feature or bug, we try to correct previous mistakes we've made, especially when it's quick to do so:

In this case, the `non_flag_args` is impressively uninformative.  Let's rename to `test_subdirectory` or similar.